substrings.rb

```diff
@@ -0,0 +1,10 @@
+def substrings(string, substrings)
+  string.downcase!
+  substrings.each_with_object({}) do |substring, occurrences_count_by_substring|
+    substring_occurrences = string.scan(/#{substring.downcase}/).length
```

Comment:
still more readable to move to the "downcase" to it's own map
it didn't work for you before because you did it inside `each_with_object` and not `map`
```suggestion
  substrings.map(&:downcase).each_with_object({}) do |substring, occurrences_count_by_substring|
    substring_occurrences = string.scan(/#{substring}/).length
```

Reply:
done